scripts/tatoeba/README.md

```diff
@@ -19,7 +19,7 @@ Setup transformers following instructions in README.md, (I would fork first).
 git clone git@github.com:huggingface/transformers.git
 cd transformers
 pip install -e .
-pip install pandas
+pip install pandas GitPython wget
```

Comment:
You're not actually using `GitPython`, right?

Reply:
it's used by `convert_marian_to_pytorch`

https://github.com/huggingface/transformers/blob/935e346959e81aa96a772c11d05734ea652932d1/src/transformers/models/marian/convert_marian_to_pytorch.py#L181